honeybee_schema/radiance/modifierset.py

```diff
@@ -0,0 +1,173 @@
+"""ModifierSet Schema"""
+from pydantic import Field, constr
+from typing import List, Union
+
+from .._base import IDdBaseModel
+from .modifier import ModifierBase, Void
+
+
+class BaseModifierSet(IDdBaseModel):
+    """Base class for the sets assigned to Faces."""
+
+    # TODO: What to use for default? Void or None?
+    exterior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A radiance modifier object for faces with an Outdoors boundary '
+                        'condition.'
+        )
+    # TODO: What to use for default? Void or None?
+    interior_modifier: ModifierBase = Field(
+            default=Void(),
+            description='A radiance modifier object for faces with a boundary condition '
+                        'other than Outdoors.'
+        )
+
+
+class WallModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Walls."""
+
+    type: constr(regex='^WallModifierSet$') = 'WallModifierSet'
+
+
+class FloorModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Floors."""
+
+    type: constr(regex='^FloorModifierSet$') = 'FloorModifierSet'
+
+
+class RoofCeilingModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's roofs."""
+
+    type: constr(regex='^RoofCeilingModifierSet$') = 'RoofCeilingModifierSet'
+
+
+class ShadeModifierSet(BaseModifierSet):
+    """Set containing all radiance modifiers needed to for a radiance model's Shade."""
+
+    type: constr(regex='^ShadeModifierSet$') = 'ShadeModifierSet'
+
+
+class ApertureModifierSet(BaseModifierSet):
```

Comment:
You can't have this one inherit from `BaseModifierSet` since the `ApertureModifierSet` has no `exterior_modifier`.  You should just have it inherit from `NoExtraBaseModel`.